### DISCUSSION TITLE
Export tandoor recipies

### DISCUSSION THREAD

**Discussion by ordo01** [TandoorRecipes/recipes#3145] [Q&A] [answered]
Hi, how can I export all recipes stored in one tandoor instance.
I do have two instances of tandoor and want to consolidate them now into one instance. Exporting the full DB and importing it is not a feasible way as it would overwrite the target instance.
Importing recieps through the web ui import function doesn't work either, since tandoor doesn't understand its own parge format.
What would be the recommended way?

**Comment by smilerz** [comment]
Import/export works but has to be to/from the same version.

**Reply by ordo01** [reply]
Could you please point me to the documentation? Could only find the full DB export import for a version upgrade but no export/import to transfer data from instance A to instance B, not overwriting what is in instance B already.

**Reply by smilerz** [reply]
export is located at `/export/`

**Reply by ordo01** [reply]
Thanks, this works for the export.
Unfortunately Tandoor does not understand its own export when trying to import the JSON on the new instance.
Only the name of the recipe and the comment have been imported. All steps, the ingredients and the images are lost.

**Reply by smilerz** [reply]
> Thanks, this works for the export.
> 
> Unfortunately Tandoor does not understand its own export when trying to import the JSON on the new instance.
> 
> Only the name of the recipe and the comment have been imported. All steps, the ingredients and the images are lost.
> 
> 

Are you importing to the same version you exported from?

**Reply by ordo01** [reply]
Yes, both instances are on the same version 1.5.17.
To import I use "Import recipe" -> "Source" and past the JSON created by the export.

**Reply by ordo01** [reply, marked as answer]
Just found my mistake. I should use "Import recipe" -> "App" -> "Tandoor"
This works seamless.
